newdle/templates/base_email.html

```diff
@@ -0,0 +1,48 @@
+<table align="center" border="0" cellpadding="0" cellspacing="0" width="100%"
+    style="margin-top: 20px; line-height: 1.7; color: #555;">
+    <tr>
+        <td>
+            <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%"
+                style="max-width: 660px; font-family: Helvetica, Arial, sans-serif; font-size: 14px; background: #FFFFFF;">
+                <tr>
+                    <td style="border: 1px solid #ddd;">
+                        <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%"
+                            style="border-collapse: collapse;">
+                            <tr>
+                                <td style="padding: 20px 20px 10px; text-align:left;">
+                                    <img src="https://raw.githubusercontent.com/indico/newdle/master/newdle/client/src/images/logo_color.svg"
+                                        width="130px"
+                                        style="opacity: 0.5; display:block; margin-left: auto; margin-right: auto;"
+                                        alt="">
+                                </td>
+                            </tr>
+                            <tr>
+                                <td>
+                                    <table border="0" cellpadding="0" cellspacing="0"
+                                        style="padding: 20px 50px 0; text-align: center; margin: 0 auto">
+                                        <tr>
+                                            <td style="border-bottom: 1px solid#ddd; padding: 0 0 20px;">
+                                                <h2 style="font-weight: normal; margin-top: 10px;">{{ title }}</h2>
+                                                {% block content %}
+                                                {% endblock %}
+                                                <br />
+                                            </td>
+                                        </tr>
+                                    </table>
+                                </td>
+                            </tr>
+                </tr>
+                <tr>
+                    <td style="text-align: center;color: #AAA; font-size: 11px; padding-bottom: 10px;">
+                        <p style="padding: 0 50px;"><br>
+                            This email was sent by <a href="{{ newdle_link }}">newdle</a>.
+                        </p>
+                    </td>
+                </tr>
+            </table>
+        </td>
+    </tr>
+</table>
+</td>
+</tr>
+</table>
```

Comment:
- missing the `<body>` tag around everything
- no trailing linebreak
- we use 2-space indentation in html files